easybuild/framework/easyconfig/tweak.py

```diff
@@ -1059,6 +1060,34 @@ def map_easyconfig_to_target_tc_hierarchy(ec_spec, toolchain_mapping, targetdir=
     return tweaked_spec
 
 
+def list_deps_versionsuffixes(ec_spec):
+    """
+    Take an easyconfig spec, parse it, extracts the list of version suffixes used in its dependencies
+
+    :param ec_spec: location of original easyconfig file
+
+    :return: The list of versionsuffixes used by the dependencies of this recipe
+    """
+    # Fully parse the original easyconfig
+    parsed_ec = process_easyconfig(ec_spec, validate=False)[0]['ec']
+
+    versionsuffix_list = []
+    for key in DEPENDENCY_PARAMETERS:
+        # loop over a *copy* of dependency dicts (with resolved templates);
+
```

Comment:
These comments are a bit off, remove altogether or update

Reply:
oops, indeed, that's a copy from another function. Removed.